h2o-py/h2o/model/segment_models.py

```diff
@@ -0,0 +1,76 @@
+# -*- encoding: utf-8 -*-
+"""
+H2O Segment Models.
+
+:copyright: (c) 2020 H2O.ai
+:license:   Apache License Version 2.0 (see LICENSE for details)
+"""
+from __future__ import absolute_import
+
+from h2o.base import Keyed
+from h2o.frame import H2OFrame
+from h2o.expr import ExprNode
+from h2o.expr import ASTId
+
+__all__ = ("H2OSegmentModels", )
+
+
+class H2OSegmentModels(Keyed):
+    """
+    Collection of H2O Models built for each input segment.
+
+    H2OFrame is similar to pandas' ``DataFrame``, or R's ``data.frame``. One of the critical distinction is that the
```

Comment:
looks like most of this doc is copied from H2OFrame...

Reply:
yes :) don't know how it happened - will fix!